lambda/src/main/resources/db/migration/V15__add-consignment-seq-and-reference.sql

```diff
@@ -0,0 +1,15 @@
+-- Create sequence that will be the consignment sequence ID used to encode b25 section of consignment reference
+CREATE SEQUENCE "ConsignmentSequenceID"
+NO CYCLE
+START WITH 1
+INCREMENT BY 1;
+
+-- Alter Consignment table to add column to store sequence ID and another for full consignment reference
+ALTER TABLE "Consignment"
+ADD COLUMN "ConsignmentSequence" bigint DEFAULT nextval('"ConsignmentSequenceID"'),
+ADD COLUMN "ConsignmentReference" text,
+ADD UNIQUE ("ConsignmentSequence", "ConsignmentReference");
+
+-- Alter consignment table to create consignment reference for existing Consignment rows
+UPDATE "Consignment"
+SET "ConsignmentReference"='TDR-2020-OLD-' || "ConsignmentSequence";
```

Comment:
Can you post on Slack to let other people know we're planning to use "TDR-2020-OLD-1234" to back-fill these values? Hopefully that's fine because it will only affect data in intg and staging, not production.

Reply:
Cheers for doing that. 👍